packages/keepThisFolderHere

```diff
@@ -0,0 +1 @@
+Keep this folder b/c the nuget config seems to depend on it.
```

Comment:
This is weird. We have this in an upper shared folder, which is where I would expect it to look.

Reply:
There is this nuget.config file in the root directory:
```
<?xml version="1.0" encoding="utf-8"?>
<configuration>
    <packageSources>
        <add key="LocalPackages" value=".\packages" />
    </packageSources>
</configuration>
```

Which in Rider resolves like this:
![image](https://github.com/dymaptic/GeoBlazor-Samples/assets/1805118/9c2c555f-9298-4a6a-b6f1-010fe13ea142)

I'm happy to delete this file if it's not helping, I thought I was helping since without this folder nothing would build for me.

Reply:
Sorry, I guess I misread, I thought this was inside your particular sample directory. Yes, I currently use that folder, I didn't realize I wasn't committing it. It's a convenient place to throw beta builds of GeoBlazor while I'm working.